app/src/main/resources/db/migration/statistikk_model/V12__legg_til_kolonne_gruppering_id.sql

```diff
@@ -0,0 +1,3 @@
+alter table notifikasjon
```

Comment:
disse to migreringene kunne sikkert vært gjort i en fil, men dersom dette allerede er deployet trenger/bør det ikke endres på.